docs/OV_Runtime_UG/automatic_batching.md

```diff
@@ -0,0 +1,40 @@
+# Automatic Batching Plugin {#openvino_docs_OV_UG_Automatic_Batching}
+
+## (Automatic) Batching Plugin Execution (C++)
+
+@sphinxdirective
+.. raw:: html
+
+    <div id="switcher-cpp" class="switcher-anchor">C++</div>
+@endsphinxdirective
```

Comment:
I suppose the content of your page is mostly language independent. Do we need separation between C++ and python?

Reply:
the new config names are C++ e.g. ov::hint::performance_mode (shall we put the string names to accomodate the Python)